### ISSUE TITLE
Equipment sets not sorting

### ISSUE THREAD

**Issue by Weetbix813** [doadin/Baggins#76]
I'm using the most current retail version, Equipment Sets are not sorting into designated sections, there is no error message, the category just doesn't appear, all armor/weapons remain in the general armor/weapons sections

**Comment by doadin** [comment]
As much as I hate to blame something else I think this is a second instance of the game api not returning proper info.
Running
for bag=0,NUM_BAG_SLOTS do
   for slot=1,C_Container.GetContainerNumSlots(bag) do
      local item = C_Container.GetContainerItemID(bag,slot)
      if not item then break end
      local inset, setstring = C_Container.GetContainerItemEquipmentSetInfo(bag, slot)
      local name = GetItemInfo(item)
      print(name, " ", inset)
   end
end

Basically if you don't know code if I loop through all bag items and check for if it is in a set(inset from above code) it is returning false for all items even though I know I have items that are part of a equipment set in my bags.

**Comment by Weetbix813** [comment]
Hmm I'm not a coder so I don't fully understand, but I know AdiBags that i switched from for other issues was still able to sort equipment sets so i'm not sure if they have it coded differently or maybe I somehow have something else thats interfering?

**Comment by doadin** [comment]
I added a fix at least for now. This fix is included in v4.5.8 .
https://github.com/doadin/Baggins/commit/5a13827f7767c8e91e4f388fc9233eccc156e0ee

**Comment by Weetbix813** [comment]
Thank you so much 😊

Dr. Rochelle M. Burnaford
She/her pronouns<http://www.transstudent.org/pronouns101/>
Psychology Area Convener 2022-2023
Professor of Psychology
Diablo Valley College

"I am not free while any woman is unfree,
even when her shackles are very different
from my own."
-Audre Lorde
